### PR TITLE
🚀 [Ads] Update Yandex & ADFOX insert codes

### PR DESCRIPTION
**Pull request by mishanga**
Yandex updated ads codes:

- https://yandex.ru/adv/news/zagruzka-reklamy-na-sayte-stala-bystree-s-novymi-kodami-vstavki-blokov-rsya-i-adfox
- https://yandex.ru/support/adfox-sites/codes/change-code.html
- https://yandex.ru/support/partner2/web/products-rtb/partner-code.html#partner-code__new-code

### CODE REVIEW

**Comment by mishanga** [comment]
plz review @DiMurer

File: ads/vendors/adfox.js [hunk 1]
```diff
@@ -13,65 +13,38 @@
...
-function checkLoading(global, data, onRender, onError) {
-  if (data.bundleName === 'banner.direct') {
-    const dblParams = {
-      blockId: data.bundleParams.blockId,
-      data: data.bundleParams.data,
-      onRender,
-      onError,
-    };
-
-    yandex(global, dblParams);
```
Comment (mishanga) [inline ads/vendors/adfox.js:69]: Using yandex is not necessary anymore. `context.js` loader can render both RTB and ADFOX ads.

**Review by powerivq — APPROVED** [review]

**Comment by mishanga** [comment]
> Please fix the conflict.

@powerivq fixed + tested

Old and new urls returns same content now:
https://yastatic.net/pcode/adfox/loader.js
https://an.yandex.ru/system/context_amp.js
https://yandex.ru/ads/system/context.js

Screenshot 2021-09-22 at 20 59 03

**Comment by DiMurer** [comment]
👍

**Comment by mishanga** [comment]
@powerivq can you merge this PR, please?

**Comment by mishanga** [comment]
@calebcordry @samouri maybe you, guys? :)

**Pull request merged**